attributes.yml

```diff
@@ -0,0 +1 @@
+command_path: '/bin/getfacl'
```

Comment:
Should this be: `acl_relative_path` ?

Reply:
will change both attribute and ruby variable to be command_relative_path